rust/interpreter/src/object.rs

```diff
@@ -0,0 +1,24 @@
+use std::collections::HashMap;
+
+use crate::value::JSValue;
+
+#[derive(Debug)]
+pub struct Object {
+	properties: HashMap<String, JSValue>
+}
+
+impl Object {
+    pub fn new() -> Self {
+        Object {
+            properties: HashMap::new(),
+        }
+    }
+
+    pub fn set(&mut self, name: String, value: JSValue) {
```

Comment:
Does rustc not warn about these methods being unused? No need to fix every patch, as long as the final build has no warnings.